tests/MongoDB.Driver.Core.TestHelpers/XunitExtensions/RequirePlatform.cs

```diff
@@ -78,7 +78,7 @@ public RequirePlatform SkipWhen(SupportedOperatingSystem operatingSystem, params
         {
             var currentOperatingSystem = GetCurrentOperatingSystem();
             var currentTargetFramework = GetCurrentTargetFramework();
-            if (operatingSystem == currentOperatingSystem && (targetFrameworks == null || targetFrameworks.Contains(currentTargetFramework)))
+            if (operatingSystem == currentOperatingSystem && ((targetFrameworks?.Length ?? 0) == 0 || targetFrameworks.Contains(currentTargetFramework)))
```

Comment:
nit: Optional suggestions for your judgment:
1. targetFrameworks?.Any() != true  or 
2. add CollectionsExtenstions.AnySafe<T>(this IEnumerable<T> enumerable) => enumerable?.Any() == true; or/and
CollectionsExtenstions.EmptySafe<T>(this IEnumerable<T> enumerable) => enumerable?.Any() != true;

Reply:
thanks for the suggestions, this could be implemented in different ways, but I don't see a significant difference between them, so I would leave this as is :)